.github/workflows/overall-tests.yml

```diff
@@ -7,6 +7,9 @@ on:
     branches: [ develop, dev/*, release/* ]
   workflow_dispatch:
 
+permissions:
+  pull-requests: write
+
```

Comment:
I don't think that was the issue. Was it?
I believe the permission was granted directly at the GitHub organization level. I am not fully certain, though.

Reply:
All right. Thx.